src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -61,7 +66,7 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "Manufacturer{"
+        return "ManufacturerService{"
```

Comment:
The class name in the toString method should be 'Manufacturer' instead of 'ManufacturerService'.